frame/pablo/src/lib.rs

```diff
@@ -75,21 +76,28 @@ pub mod pallet {
 			fee: Permill,
 			protocol_fee: Permill,
 		},
+		ConstantProduct {
+			pair: CurrencyPair<AssetId>,
+			fee: Permill,
+			owner_fee: Permill,
+		},
 	}
 
 	#[derive(RuntimeDebug, Encode, Decode, MaxEncodedLen, Clone, PartialEq, Eq, TypeInfo)]
 	pub enum PoolConfiguration<AccountId, AssetId> {
 		StableSwap(StableSwapPoolInfo<AccountId, AssetId>),
+		ConstantProduct(ConstantProductPoolInfo<AccountId, AssetId>),
 	}
 
 	type AssetIdOf<T> = <T as Config>::AssetId;
 	type BalanceOf<T> = <T as Config>::Balance;
 	type AccountIdOf<T> = <T as frame_system::Config>::AccountId;
 	type PoolIdOf<T> = <T as Config>::PoolId;
-	type PoolConfigurationOf<T> =
+	pub type PoolConfigurationOf<T> =
```

Comment:
Is this `pub` required ?

Reply:
Good catch, it was from an earlier version